runtime/jilgen/module.xml

```diff
@@ -59,6 +59,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<library name="j9pool" type="external"/>
 			<library name="j9exelib"/>
 			<library name="j9utilcore"/>
+			<library name="omrsig">
```

Comment:
Can you explain why this is necessary?  It's surprising to see this linked to the signal chaining library

Reply:
Removed.